deploy/ansible/roles/asset-registry/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+- name: Create directory
+  file:
+    path: /opt/asset-registry
+    state: directory
+    mode: '0755'
+
+
+- name: Template docker-compose.yml
+  template:
+    src: docker-compose.yml.j2
+    dest: /opt/asset-registry/docker-compose.yml
+    mode: '0644'
+
+- name: Stop existing services gracefully
+  community.general.docker_compose:
+    project_src: /opt/asset-registry
+    timeout: 180
+    state: present
+    stopped: yes
+
+- name: Tear down existing services
```

Comment:
This step will also try to stop the services before tearing the resources down, but I am fine with the two-steps thing!

Reply:
Hm, I didn't know that. 
TBH, I took that 2-steps approach from the chrysalis infrastracte repo you shared with me.